includes/admin/class-wc-payments-admin.php

```diff
@@ -204,6 +204,15 @@ public function register_payments_scripts() {
 		$error_message = get_transient( WC_Payments_Account::ERROR_MESSAGE_TRANSIENT );
 		delete_transient( WC_Payments_Account::ERROR_MESSAGE_TRANSIENT );
 
+		// Prepare currency-related information, which will be added to JS settings object.
+		$default_currency      = $this->account->get_default_store_currency();
+		$supported_currencies  = $this->account->get_supported_store_currencies();
+		$currency_code_options = get_woocommerce_currencies();
+		foreach ( $supported_currencies as $code ) {
+			$code_uppercase           = strtoupper( $code );
+			$localized_names[ $code ] = sprintf( '%s (%s)', $currency_code_options[ $code_uppercase ] ?? $code, $code_uppercase );
+		}
```

Comment:
we might not need this anymore, at least for filter.

Reply:
Reverted in 1889eed2c444b50617632e02bf802f564a18743e